src/qr.jl

```diff
@@ -19,3 +19,246 @@ function qr(A::StaticMatrix, pivot::Type{Val{false}}; thin::Bool=true)
     RT = similar_type(A, T)
     QT(Q0), RT(R0)
 end
+function qr(A::Size{(1,1)}, pivot::Type{Val{false}})
```

Comment:
This signature can't be right... `A` should be a `StaticMatrix` and you should have a thunk to generate a `Size` and dispatch to a 3-argument e.g. `_qr` function.